src/leader_cron_task.erl

```diff
@@ -495,6 +495,17 @@ extract_integers(Spec, Min, Max, Acc) ->
 	     end,
     extract_integers(T, Min, Max, [Values|Acc]).
 
+sleep_accounting_for_max(TimeInMillis) ->
+       case (TimeInMillis > 100000000) of 
```

Comment:
Can you `-define(SLEEP_CHUNK_SIZE, 100000000).` and use that in place of the various `100000000`s? It _looks_ like the same number of `0`s everywhere, but best to be sure.

Do you know what the maximum value is (or where it is defined)?